test/bases/renku_data_services/data_api/test_sessions.py

```diff
@@ -80,11 +80,20 @@ async def test_get_session_environment(
 
 
 @pytest.mark.asyncio
-async def test_post_session_environment(sanic_client: SanicASGITestClient, admin_headers) -> None:
+@pytest.mark.parametrize(
+    "image_name",
+    [
+        "renku/renku",
+        "u/renku/renku:latest",
+        "docker.io/renku/renku:latest",
+        "renku/renku@sha256:eceed25752d7544db159e4144a41ed6e96e667f39ff9fa18322d79c33729a18c",
```

Comment:
I know it is probably fine. But lets add one or more gitlab repo image names here. Just to be safe. Because a large chunk of our images are from gitlab image repos.